attributes/default.rb

```diff
@@ -75,9 +75,15 @@
   default['openldap']['packages']['client_pkg'] = 'ldap-utils'
   default['openldap']['packages']['srv_pkg'] = 'slapd'
 when 'rhel'
-  default['openldap']['packages']['bdb'] = 'db4-utils'
-  default['openldap']['packages']['client_pkg'] = 'openldap-clients'
-  default['openldap']['packages']['srv_pkg'] = 'openldap-servers'
+  if node['platform'] == 'centos' && node['platform_version'].to_i >= 7
+    default['openldap']['packages']['bdb'] = 'libdb-devel'
```

Comment:
Since this is only attribute that differs the others can be pushed outside this if statement.